File: Atrasti.ReactNative/Atrasti/src/Api/Models/Responds/ManageProfile_Res.ts

import { Company_Res } from './Company_Res';
import { CompanyInfo_Res } from './CompanyInfo_Res';
import { BaseCategory_Res } from './BaseCategory_Res';

export interface ManageProfile_Res {
    company: Company_Res;
    companyInfo: CompanyInfo_Res;
    usedCategories: BaseCategory_Res[];
    allCategories: BaseCategory_Res[];
}

File: Atrasti.API/Models/Profile/ManageProfile_Res.cs
using System.Collections.Generic;
using Atrasti.API.Models.Categories;
using Atrasti.API.Models.Company;

namespace Atrasti.API.Models.Profile
{
    public class ManageProfile_Res
    {
        public Company_Res Company { get; set; }
        public CompanyInfo_Res CompanyInfo { get; set; }
        public IList<BaseCategory_Res> UsedCategories { get; set; }
        public IList<BaseCategory_Res> AllCategories { get; set; }
    }
}